include/Core/EmailLog.php

```diff
@@ -65,6 +65,14 @@ class EmailLog {
 	public $table_manager;
 
 	/**
+	 * Email Log Plugins Setting.
+	 *
+	 * @since 2.1.0
+	 * @var \EmailLog\Core\UI\Setting\Setting
+	 */
+	public $setting;
```

Comment:
Are we using this variable anywhere? If not let's add the setting as a loadie and remove this.

Reply:
Fixed.